### ISSUE TITLE
`filter` property is missing

### ISSUE THREAD

**Issue by surma** [GoogleChrome/css-triggers#10]

**Comment by rachsmithcodes** [comment]
in the same vein, it would be great to get the color legend back for Layout, Paint, Composite back as well. 
![screenshot 2016-02-22 11 09 25](https://cloud.githubusercontent.com/assets/1632086/13229293/0926c284-d955-11e5-8f33-2080ac4f3d5f.png)

Thanks so much for this resource!

**Comment by Elindorath** [comment]
I think he rather talked about the css property named `filter`.

But I support your idea too, it would be much clearer.

**Comment by tkmoney** [comment]
+1

**Comment by DGizmo** [comment]
+1

**Comment by tvler** [comment]
+1

**Comment by surma** [comment]
The search bar (for *filtering* the list of properties has been added to csstriggers.com :)

**Comment by brianblakely** [comment]
Started to test this (in case no one else if working on it), but I have a question.

This is vendor-prefixed in Blink, but not Edge/Gecko/Webkit.  As such, should there be two entries in the matrix, one for `-webkit-filter` and one for `filter`?

**Comment by surma** [comment]
I don’t think we are using prefixes anywhere else so I’d say just call it `filter` for all browsers. 

I assume @paullewis will stop me if I’m wrong ;)

**Comment by polarathene** [comment]
Any progress on this? I've been asked for a PR involving `filter: blur()` if it has perf/battery problems based on many articles between 2012-2016 on mobile battery life and perf issues. Pretty sure mobile hardware has come a long way even for low end devices, along with software(OS and browsers) improvements. Majority accelerate blur via GPU now right?

Any advice on how to profile?

**Comment by Jab2870** [comment]
did anyone find out whether the filter property would require the browser to re-paint.

**Comment by vkrbt** [comment]
@Jab2870 yes, `filter` require browser repaint

**Comment by vkrbt** [comment]
Could `filter` property be added to list? Looks like property supported by all browser engines

**Comment by Jared-Dev** [comment]
I see a comment noting that a re-paint is required. Does `transition-property: filter;` require a re-flow as well?
